src/tinymce.js

```diff
@@ -85,12 +102,13 @@ angular.module('ui.tinymce', [])
 
             // Update model when an object has been resized (table, image)
             ed.on('ObjectResized', function() {
-              ed.save();
-              updateView(ed);
+              scope.$emit('tinymce:ObjectResized',ed);
             });
 
             ed.on('remove', function() {
+              element.off();
```

Comment:
Not sure this is really needed since we are calling remove on the next line. I believe remove will also take the action of .off(), no?